crowbar_framework/app/controllers/api/crowbar_controller.rb

```diff
@@ -19,6 +19,15 @@ class Api::CrowbarController < ApiController
 
   api :GET, "/api/crowbar", "Show the crowbar object"
   api_version "2.0"
+  example '
+  {
+    "version": "4.0",
```

Comment:
I'd prefer a long line in the example than a comment on a github review ;)
You can also add those lines outside the json example and make it multiline... 

I'm reading Apipie issues and it seems there is no better approach to represent the responses other than examples. A lot of people reference to swagger.io or raml.org, claiming they both support it...